### PR TITLE
fix(@vtmn/react): add missing onChange event to `VtmnDropdown`

### PR DESCRIPTION
**Pull request by 0x2A-git**
## Changes description

Injected new event in VtmnDropdown's children and then called children's callback.
This event was missing according to the showcase : https://www.decathlon.design/726f8c765/v/0/p/42d824-dropdown/b/28931f

## Context

This P.R aims to resolve issue #1377 by adding an onChange event to VtmnDropdown component.

## Checklist

- [X] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [X] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [X] Check your code additions will fail neither code linting checks.
- [X] I have reviewed the submitted code.
- [X] I have tested on related showcases.
- [X] If it includes design changes, please ask for a review `design-system-core-team-design` GitHub team.

## Does this introduce a breaking change?

- No

### CODE REVIEW

File: packages/sources/react/src/components/actions/VtmnDropdown/VtmnDropdown.tsx [hunk 1]
```diff
@@ -46,6 +55,20 @@ export const VtmnDropdown = ({
...
+              ? React.cloneElement(child, {
+                  ...child.props,
+                  onChange: (ev: React.ChangeEvent<HTMLDivElement>) => {
+                    onChange && onChange(child.props.value);
+                    child.props.onChange && child.props.onChange(ev);
+                  },
+                })
+              : child,
+          )}
           {children}
```
Comment (lauthieb) [inline packages/sources/react/src/components/actions/VtmnDropdown/VtmnDropdown.tsx:72]: Why do we still keep this line `{children}` if you already map on children thanks to `React.Children.map`?
Thanks for your help!

**Review by lauthieb — COMMENTED** [review]
Thanks @0x2A-git, just one question :)

**Review by lauthieb — APPROVED** [review]
Thanks! Now that's ok for me.
@Decathlon/design-system-core-contrib-web-dev can you please review?

**Comment by 0x2A-git** [comment]
> Thanks! Now that's ok for me.
> @Decathlon/design-system-core-contrib-web-dev can you please review?

Nice ! Thanks for having pointed out this issue I didn't notice it when testing it :S

Good afternoon :D

**Review by lauthieb — APPROVED** [review]

**Pull request merged**